src/text/isMatch.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Test a string against regexp
+ * @param {string} target
+ * @param {string} regExp
+ * @return Boolean
+ */
+
+export const isMatch = (target: string, rex: string | RegExp): Boolean => {
+  return new RegExp(rex).exec(target) !== null;
```

Comment:
JavaScript and GeneXus' regular expressions use the same format? Assuming that `rex` comes from GeneXus code, doesn't it need some kind of pre-preocessing?

Reply:
I think not, because the javascript regExp are compatible with GX spec